index.html

```diff
@@ -768,6 +768,87 @@ <h2>Verifiable Credential</h2>
           <pre data-include="./examples/vc-2.0/verifiable-credential.jwt" data-include-format="text"></pre>
         </aside>
       </section>
+
+    <section>
+      <h2>Mapping to the Verifiable Credentials Data Model</h2>
+      <p>
+We define the following mapping from <code>vc+jwt</code> to
+<code>credential+ld+json</code>:
+      </p>
+      <ul>
+        <li>
+Extract <code>iss</code>, <code>sub</code>, <code>iat</code>, <code>nbf</code>,
+<code>exp</code>, <code>jti</code>, and <code>aud</code> as registered claims.
+        </li>
+        <li>
+Set aside all other claims as subject claims.
+        </li>
+        <li>
+To construct the VCDM-complaint JSON:
+          <ul>
+            <li>
+Set the value of <code>@context</code> to be
+<code>"https://www.w3.org/ns/credentials/v2"</code>.
+            </li>
+            <li>
+Set the value of <code>type</code> to be <code>["VerifiableCredential"]</code>.
```

Comment:
This implies that a JWT VC with a `type` property will be mapped to `credentialSubject.type` and that the VCDM representation of it will only ever have the value `["VerifiableCredential"]` for the `type`.

Is this the intent here, or am I missing something?

(same logic applies to `@context`)

Reply:
That is correct, if the issue to of the JWT wanted to produce `credential+ld+json` or `vc+ld+json` there are instructions for how to do that in this document.